contracts/src/v0.8/ccip/test/rmn/RMNRemoteSetup.t.sol

```diff
@@ -121,7 +114,12 @@ contract RMNRemoteSetup is BaseTest {
       )
     );
     (uint8 v, bytes32 r, bytes32 s) = vm.sign(wallet, digest);
-    return (v, IRMNRemote.Signature({r: r, s: s}));
+    // RMNRemote only supports sigs with v=27, so adjust if necessary
+    if (v == 28) {
+      uint256 N = 0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFEBAAEDCE6AF48A03BBFD25E8CD0364141;
```

Comment:
can you add a comment on what this actually does and why we're adjusting S when v is 28?